Woof/Woof/ViewModel/LoginViewModel.swift

```diff
@@ -4,25 +4,20 @@ import Foundation
 final class LoginViewModel: ObservableObject {
     // MARK: - Internal interface
 
-    /// Indicates whether the owner role is selected.
-    ///
-    /// Returns `true` if the owner role is selected, otherwise `false`.
-    ///
-    /// The property publishes updates when the value is changed.
-    var isOwnerRoleSelected = false {
-        didSet {
-            if !isCurrentOwnerExists {
-                saveNewOwner()
-            }
-        }
+    /// Indicates whether to show owner onboarding flow.
+    @Published var shouldShowOwnerOnboarding = false
+    /// Indicates whether to show sitter onboarding flow.
+    @Published var shouldShowSitterOnboarding = false
+
+    /// Performs actions triggered by the owner role choice.
+    func ownerFlowDidSelected() {
```

Comment:
The name of the function is not clear to me. I expected that it would contain all business logic related to owner flow, but it just checked if we needed to present the onboarding view.

Reply:
The current function just handles the process of displaying the onboarding view based on the owner/sitter existing.
How about ` func handleOwnerOnboarding()`?

Reply:
The current name refer to kind of a delegate method. For example: `keyboardDidHideNotification`, `keyboardWillShowNotification`. Something like these.
The name describes when the function will be called and it's meaningful for this function. I don't understand what the `handleOwnerOnboarding` name gives.